composer.json

```diff
@@ -16,7 +16,7 @@
     "require": {
         "php": "^8.0",
         "consolidation/robo": "^4.0",
-        "symfony/finder": ">=2.7 <=6.0",
```

Comment:
symfony/finder constraint should match Codeception 5: `"symfony/finder": ">=4.4.24 <7.0"` unless you don't support 6.1 yet.